### ISSUE TITLE
simctl getDevices is failing the regexp with versionMatchRe.match is not a function

### ISSUE THREAD

**Issue by ChrisEdgington** [appium/appium#12205]
## The problem

When Appium is preparing to launch the simulator and my app ... I see the following in the debug log:

[debug] [simctl] Unable to get JSON device list: versionMatchRe.match is not a function
[debug] [simctl] Falling back to manual parsing

## Environment

* Appium version: 1.11.1
* Desktop OS/version used to run Appium: macOS 10.14.3
* Xcode version: 10.1 (10B61)
* Node.js version: 11.9.0
* Mobile platform/version under test: iOS 12
* Real device or emulator/simulator: sim
* Appium CLI or Appium.app|exe: cli

## Link to Appium logs

https://gist.github.com/ChrisEdgington/520d7a13aa4cee7cbc721898224bd7a7

**Comment by dpgraham** [comment]
Thanks for reporting this.

What version of Xcode are you on?

**Comment by ChrisEdgington** [comment]
Sorry I forgot that piece of info: * Xcode version: 10.1 (10B61)

**Comment by dpgraham** [comment]
I think I found the problem already. I'll have a fix in shortly.

@ChrisEdgington do you have time to test the fix for me when it's ready?

**Comment by ChrisEdgington** [comment]
Sure ... but warning, I'll start reporting the other issues I'm having too :O

**Comment by dpgraham** [comment]
Published as a release candidate

Can you try installing it via `npm install -g appium@rc`.... it will be version `1.11.2-rc.0`

**Comment by ChrisEdgington** [comment]
I can confirm that my install worked:
> Welcome to Appium v1.11.2-rc.0

However, it did not change the behavior:
```
[debug] [XCUITest] Current user: 'chris'
[debug] [simctl] Unable to get JSON device list: versionMatchRe.match is not a function
[debug] [simctl] Falling back to manual parsing
[debug] [simctl] Unable to get JSON device list: versionMatchRe.match is not a function
[debug] [simctl] Falling back to manual parsing
```

**Comment by dpgraham** [comment]
Awww snap... alright thanks. I'll have to dig into some more tomorrow morning.

Btw, I never asked. Does it still work?

**Comment by ChrisEdgington** [comment]
Yes, the manual searching appears to work fine and the app launch proceeds.

**Comment by dpgraham** [comment]
Ok, that's a relief. I'll still look into this to see why that's happening. It might not be a bug, but rather just bad logging.

**Comment by mykola-mokhnach** [comment]
The issue has been fixed a while ago in https://github.com/appium/node-simctl/pull/75

**Comment by dpgraham** [comment]
@ChrisEdgington can you try again... I just published another rc `v1.11.2-rc.1` (`npm install -g appium@rc`). This one has the updated library.

**Comment by ChrisEdgington** [comment]
@dpgraham rc.1 definitely does resolve the issue, the simctl messages are no longer present in my log.
